prototypes/ScheduledMergesTestQLS.hs

```diff
@@ -222,10 +247,19 @@ instance InLockstep Model where
           | not (null kvars)
           ]
        ++ [ (1, fmap Some $
-                  ADump <$> elements vars)
+                  ADuplicate <$> elements vars)
           ]
        ++ [ (1, fmap Some $
-                  ADuplicate <$> elements vars)
+                  AUnion <$> elements vars
+                         <*> elements vars)
+          ]
+          -- TODO: only supply to tables with unions?
```

Comment:
It would require tracking a union/non-union flag per table as part of the model, and then it should be relatively straight-forward, or how do you envision it?